solutions/src/main/java/ru/sberbank/school/task07/DifferentWordsImpl.java

```diff
@@ -0,0 +1,27 @@
+package ru.sberbank.school.task07;
+
+import lombok.NonNull;
+
+import java.io.FileNotFoundException;
+import java.util.Set;
+import java.util.TreeSet;
+
+public class DifferentWordsImpl implements DifferentWords<Set<String>> {
+    private FileParser parser;
+
+    public DifferentWordsImpl(@NonNull FileParser fileParser) {
+        this.parser = fileParser;
+    }
+
+    @Override
+    public Set<String> findSortedDifferentWords(String pathToFile) throws FileNotFoundException {
+        Set<String> set = new TreeSet<>((o1, o2) -> o1.length() == o2.length() ?
+            o1.compareTo(o2) : o1.length() - o2.length());
```

Comment:
&#39;o1&#39; has incorrect indentation level 12, expected level should be 16.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a1392879570>*